hedera-node/hedera-token-service-impl/src/main/java/module-info.java

```diff
@@ -15,6 +15,8 @@
     requires com.swirlds.jasperdb;
     requires org.apache.commons.lang3;
     requires org.slf4j;
+    requires tuweni.bytes;
+    requires org.jetbrains.annotations;
```

Comment:
Should use findbugs instead of jetbrains for annotations

Reply:
updated